lib/rules/time-min-milliseconds/index.js

```diff
@@ -62,47 +62,44 @@ const rule = (primary, secondaryOptions) => {
 			}
 
 			if (shorthandTimeProperties.has(propertyName)) {
-				const valueListList = postcss.list.comma(propertyValue);
-
-				for (const valueListString of valueListList) {
-					const valueList = postcss.list.space(valueListString);
-
-					if (ignoreDelay) {
-						// Check only duration time values
-						const duration = getDuration(valueList);
-
-						if (duration && !isAcceptableTime(duration)) {
-							complain(decl, propertyValue.indexOf(duration), duration.length);
-						}
-					} else {
-						// Check all time values
-						for (const value of valueList) {
-							if (!isAcceptableTime(value)) {
-								complain(decl, propertyValue.indexOf(value), value.length);
-							}
-						}
-					}
-				}
+				const parsedValue = valueParser(getDeclarationValue(decl));
+				let isValueTimeCount = 0;
```

Comment:
[suggest] The `is*` prefix often indicates a boolean variable and so the `isValueTimeCount` name seems a bit confusing to me. How about using `timeValueCount` instead?

```suggestion
				let timeValueCount = 0;
```